tools/actions/create-release.sh

```diff
@@ -0,0 +1,36 @@
+#!/bin/sh
+
+set -xe
+
+RELEASE_DATE=$1
+RELEASE_LINE=$2
+
+if [ -z "$RELEASE_DATE" ] || [ -z "$RELEASE_LINE" ]; then
+  echo "Usage: $0 <RELEASE_DATE> <RELEASE_LINE>"
+  exit 1
+fi
+
+git config --local user.email "github-bot@iojs.org"
+git config --local user.name "Node.js GitHub Bot"
```

Comment:
Can we put this in the GHA workflow file instead, so I can run it locally without affecting my workspace?